src/engine/platform/c140.h

```diff
@@ -0,0 +1,115 @@
+/**
+ * Furnace Tracker - multi-system chiptune tracker
+ * Copyright (C) 2021-2023 tildearrow and contributors
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#ifndef _C140_H
+#define _C140_H
+
+#include "../dispatch.h"
+#include "sound/c140.h"
+#include "../fixedQueue.h"
+
+class DivPlatformC140: public DivDispatch {
+  struct Channel: public SharedChannel<int> {
+    struct VolumeUpdate {
```

Comment:
Get rid of this struct and put two bool variables (volChangedL and volChangedR) instead.

Reply:
I am doing this - please don't